plugin/cache/item.go

```diff
@@ -111,9 +111,9 @@ func (i *item) ttl(now time.Time) int {
 	return ttl
 }
 
-// filterDNSSEC filters the dns.RR slice and removes DNSSEC records. This removes:
-// NSEC,NSEC3,DS and RRSIG/SIG records. DNSKEYs are left alone on the assumptions that if they are
-// there the client explictly asked for them.
+// isDNSSEC return true id r is a DNSSEC record. NSEC,NSEC3,DS and RRSIG/SIG
+// are DNSSEC records. DNSKEYs are left alone on the assumptions that if they
+// are there the client explictly asked for them.
```

Comment:
"DNSKEYs are left alone ..."  sentence doesn't apply to this function (but would to a "filter" function)

Reply:
done